includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -173,7 +173,7 @@ public function init_fields() {
 					'label'       => __( 'Job type', 'wp-job-manager' ),
 					'type'        => $job_type,
 					'required'    => true,
-					'placeholder' => '',
+					'placeholder' => __( 'Choose job type&hellip;', 'wp-job-manager' ),
```

Comment:
Adding the HTML entity `&hellip;` into the translation might make translations more difficult, as translators need to know what this means AND it is inconsistent, as no other placeholders in this form use them.

I would suggest ditching the `&hellip;` but at bare minimum, move them outside of the translation like so: `__( 'Choose job type', 'wp-job-manager' ) . '&hellip;'`